includes/class-internal-events.php

```diff
@@ -148,21 +148,20 @@ public function register_internal_events_schedules( $schedules ) {
 	 * Schedule internal events
 	 */
 	public function schedule_internal_events() {
-		$when = strtotime( sprintf( '+%d seconds', JOB_QUEUE_WINDOW_IN_SECONDS ) );
```

Comment:
it seems a bit more readable to retain this and maybe just change to $when_default